installers/linux/al2/spec/java-amazon-corretto.spec.template

```diff
@@ -55,9 +55,18 @@
 %global abs2rel %{__perl} -e %{script}
 
 # Filter out private libraries so RPM does not list them as provided/required
-%global _privatelibs libjli[.]so.*
+%global _privatelibs libjli[.]so.*|libattach[.]so.*|libawt_headless[.]so.*|libawt[.]so.*|libdt_socket[.]so.*|libextnet[.]so.*|libfontmanager[.]so.*|libinstrument[.]so.*|libj2gss[.]so.*|libj2pcsc[.]so.*|libj2pkcs11[.]so.*|libjaas[.]so.*|libjavajpeg[.]so.*|libjdwp[.]so.*|libjimage[.]so.*|libjsound[.]so.*|libjsvml[.]so.*|liblcms[.]so.*|libmanagement[.]so.*|libmanagement_agent[.]so.*|libmanagement_ext[.]so.*|libmlib_image[.]so.*|libnet[.]so.*|libnio[.]so.*|libprefs[.]so.*|librmi[.]so.*|libsaproc[.]so.*|libsctp[.]so.*|libsyslookup[.]so.*|libzip[.]so.*|libsplashscreen[.]so.*|libawt_xawt[.]so.*
```

Comment:
How is this list populated/updated? Do we expect it to change in the future?

Reply:
Manually created and it doesn't change often, but it will change at some point. Missing a new private lib isn't the end of the world and it's unlikely that anyone else would add a 'requires' on it.